core/common/src/MapImagerySettings.ts

```diff
@@ -31,7 +31,17 @@ export interface MapImageryProps {
 /** Normalized representation of base layer properties -- these can be represented by either a full map layer or a simple color.
  * @beta
  */
-export type BaseLayerSettings = MapLayerSettings | ColorDef;
+export type BaseLayerSettings = BaseMapLayerSettings | ColorDef;
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export namespace BaseLayerSettings {
+  export function fromJSON(props: BaseLayerProps): BaseLayerSettings {
+    const settings = typeof props === "number" ? ColorDef.fromJSON(props) : BaseMapLayerSettings.fromJSON(props);
+
+    // ###TODO handle undefined BaseMapLayerSettings differently (what should be default?), or make MapImagerySettings.fromJSON possibly return undefined.
```

Comment:
@mdastous-bentley ?

Reply:
I asssume if someone turned ON ViewFlags.backgroundMap, he would expect to see a map appears even though he didnt customized the map settings? In that case I guess I guess it should be bing/hybrid.

BackgroundMapProvider.fromJSON and BackgroundMapSettings._provider also defaults to bing/hybrid